src/components/Calculator.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sign: '',
+      num: 0,
+      res: 0,
+    };
+  }
```

Comment:
- [ ] Kindly follow the given design template color swatches, your design differs from the given design in terms of the `UI` design, kindly fix this.

Given design
![image](https://user-images.githubusercontent.com/81932478/197901960-3fd7f0f9-0ade-4207-a27c-f6630f3e4640.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/197902045-0aaf6bc1-5843-4dc5-a3af-2262b1f7a620.png)